app/scripts/lib/rpc-method-middleware/handlers/ethereum-chain-utils.test.ts

```diff
@@ -245,15 +182,22 @@ describe('Ethereum Chain Utils', () => {
         });
         await switchChain('0x1', 'mainnet', 'approvalFlowId');
 
-        expect(mocks.requestPermissionApprovalForOrigin).toHaveBeenCalled();
+        expect(
+          mocks.requestPermittedChainsPermissionIncrementalForOrigin,
+        ).toHaveBeenCalled();
         expect(mocks.setActiveNetwork).not.toHaveBeenCalled();
         expect(end).toHaveBeenCalledWith();
       });
     });
 
     describe('with an existing CAIP-25 permission granted from the multichain flow (isMultichainOrigin: true) and the chainId is not already permissioned', () => {
```

Comment:
🤔 is it weird that we have tests for the multichain API in here already?

Reply:
yeah. To fix this we could just drop/move all of the `isMultichainOrigin` checks